test/deploy.sh

```diff
@@ -154,8 +161,15 @@ sleep 2
 # Start nodes
 i=0
 while IFS='' read -r line || [[ -n "$line" ]]; do
-  IFS=' ' read ip port mode account <<< $line
-  args=("${base_args[@]}" -ip "${ip}" -port "${port}" -key "/tmp/${ip}-${port}.key" -db_dir "db-${ip}-${port}" -accounts "${account}")
+  IFS=' ' read ip port mode account blspub <<< $line
+  if [ -f "${blspub}.key" ]
+    then
+      echo ""${blspub}.key" already in local."
+   else
+      aws s3 cp "s3://harmony-secret-keys/bls-test/${blspub}.key" .
+  fi
+
+  args=("${base_args[@]}" -ip "${ip}" -port "${port}" -key "/tmp/${ip}-${port}.key" -db_dir "db-${ip}-${port}" -accounts "${account}" -blspass_file blspass.txt -blskey_file "${blspub}.key")
```

Comment:
With `-blspass` and `utils.GetPassphraseFromSource`, it should be:
```suggestion
  args=("${base_args[@]}" -ip "${ip}" -port "${port}" -key "/tmp/${ip}-${port}.key" -db_dir "db-${ip}-${port}" -accounts "${account}" -blspass file:blspass.txt -blskey_file "${blspub}.key")
```